pyega3/libs/data_file.py

```diff
@@ -12,7 +12,7 @@
 import psutil
 from tqdm import tqdm
 
-from . import utils
+from pyega3.libs import utils
```

Comment:
I had to change all the relative imports into absolute ones, otherwise I received a ModuleNotFoundError, like this:

```
$ git co master && git fetch --all && git pull
...
$ docker run --rm -it -v $PWD:/app -w /app python:3.6 bash
root@07ca03c34f5d:/app# pip install -e .
...
root@07ca03c34f5d:/app# pyega3 --test fetch EGAF00001753741
Traceback (most recent call last):
  File "/usr/local/bin/pyega3", line 33, in <module>
    sys.exit(load_entry_point('pyega3', 'console_scripts', 'pyega3')())
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 474, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2846, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2450, in load
    return self.resolve()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2456, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/app/pyega3/pyega3.py", line 11, in <module>
    from libs import data_file
ModuleNotFoundError: No module named 'libs'
```

Reply:
I created a small [Confluence page](https://www.ebi.ac.uk/seqdb/confluence/display/EGA/Testing+the+pyega3+download+client) which contains the above commands, so that I won't forget it. If needed, I can move that document into the GitHub repository, next to the code (instead of a Confluence page).